src/main/java/com/hrznstudio/titanium/component/inventory/InventoryComponent.java

```diff
@@ -219,6 +240,71 @@ public T getComponentHarness() {
         return this;
     }
 
+    /**
+     * @param slot The slot to render the stack in
+     * @param stack The Itemstack to render in the slot
+     * @return itself
+     */
+    public InventoryComponent<T> setSlotToItemStackRender(int slot, ItemStack stack) {
+        this.slotToStackRenderMap.put(slot, stack);
+        return this;
+    }
+
+    /**
+     * @param slot
+     * @param color
+     * @return
+     */
+    public InventoryComponent<T> setSlotToColorRender(int slot, int color) {
+        if (!this.colorGuiEnabled) {
+            this.colorGuiEnabled = true;
```

Comment:
If SidedInvComp uses the Setter, this should too, same with others so everything uses the same